opendevin/core/schema/config.py

```diff
@@ -4,6 +4,7 @@
 class ConfigType(str, Enum):
     # For frontend
     LLM_CUSTOM_LLM_PROVIDER = 'LLM_CUSTOM_LLM_PROVIDER'
+    LLM_DROP_PARAMS = 'LLM_DROP_PARAMS'
```

Comment:
Why is this config needed in the frontend?

Reply:
This enum has been holding all or almost all settings, because we expect to implement them in FE at some point. Almost none of these was needed when added, they were added for future use.